shared/middleware/helpers/redis_table.rb

```diff
@@ -47,6 +48,8 @@ def initialize(redis, pub_sub_api, shard_id, table_name, expire_in = nil)
   # @return [Hash] The inserted value, including the new :id field.
   def insert(value, ignored_ip=nil)
     new_id = next_id
+    new_uuid = SecureRandom.uuid
+    value = value.merge({'uuid' => new_uuid})
```

Comment:
The latter, yeah.

It's always a challenge when you have two almost-but-not-quite parallel concepts like this. I think I like merging the ID in before setting the value, but I could be convinced otherwise. What side effects, if any, would that have?